activesupport/lib/active_support/file_evented_update_checker.rb

```diff
@@ -0,0 +1,71 @@
+require 'listen'
+  module ActiveSupport
+    class FileEventedUpdateChecker
+      attr_reader :listener
+      def initialize(files, dirs={}, &block)
+      	@files = Set.new
+      	@files = files.map { |f| File.expand_path(f)}
+      	@dirs = Hash.new
+      	dirs.map do |key,value|
```

Comment:
This can be like 
files.each do |file|
 @files << File.expand_path(f)
end
But the same will be done using map. So why to use each instead of map?